backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -224,8 +226,14 @@ private ApiUser getApiUser(UUID id, Boolean includeArchived) {
     return result;
   }
 
+  // current user is checked using their token
+  private boolean isCurrentUserAdmin() {
+    return _authService.isSiteAdmin();
+  }
+
+  // other users are checked by querying okta
```

Comment:
I don't think we can do this—at least, we can't have this method still be called in a tight loop by `getUsersInCurrentOrg`. If we run the `users` query and this calls getGroupMembers once per user, #1257 is probably going to go from "slow" to "hits Okta API rate limit and breaks hard for any large organization."

Reply:
I think`isAdmin` should be removed or always returns false when the user is not the current user. We don't have any business requirements to know if another user is a site admin and to Ben's point we can't query okta once per user

Reply:
Returning false provides a nice path toward removal.  It looks removing `isAdmin()` can cascade to removing several other admin-related fields and functions in other classes.

Reply:
Updated the PR with your suggestion and added issue https://github.com/CDCgov/prime-simplereport/issues/1298 to track removal of `isAdmin`.